src/abstractdataframe/abstractdataframe.jl

```diff
@@ -76,6 +82,53 @@ selector (this is useful in particular with regular expressions, `Cols`, `Not`,
   for columns that should be kept
 
 See also [`propertynames`](@ref) which returns a `Vector{Symbol}`.
+
+# Examples
+```jldoctest
+julia> df = DataFrame([1 3 3 2; missing 2 missing 4; missing 4 2 4], :auto);
+
+julia> disallowmissing!(df, 2)
```

Comment:
This looks simpler to me:
```suggestion
julia> df = DataFrame(x1=[1, missing, missing], x2=[3, 2, 4], x3=[3, missing, 2], x4=[2, 4, 4])
```